api/v1alpha1/databasecluster_types.go

```diff
@@ -145,13 +145,20 @@ type Proxy struct {
 	Resources Resources `json:"resources,omitempty"`
 }
 
+// BackupSource represents settings of a source where to get a backup to run restoration.
+type BackupSource struct {
+	// Path is the path to the backup file/directory.
+	Path string `json:"path"`
+	// BackupStorageName is the name of the BackupStorage used for backups.
+	BackupStorageName string `json:"backupStorageName"`
+}
+
 // DataSource is the data source configuration.
 type DataSource struct {
-	// BackupName is the name of the backup from backup location to use
-	BackupName string `json:"backupName"`
-	// BackupStorageName is the name of the BackupStorage CR that defines the
-	// storage location
-	BackupStorageName string `json:"backupStorageName"`
+	// DBClusterBackupName is the name of the DB cluster backup to restore from
+	DBClusterBackupName string `json:"dbClusterBackupName,omitempty"`
```

Comment:
Do you think that dbClusterBackupName is good enough? What about clusterBackupName? It's part of DatabaseCluster CR and databaseCluster.spec.dataSource.clusterBackupName sounds clear while databaseCluster.spec.dataSource.dbClusterBackupName sounds a bit redundant. What do you think?

Reply:
I went with this approach to keep it consistent with the `DBClusterName` we already had in the `DatabaseClusterBackup`. My reasoning was if a reference to the `DatabaseCluster` is `DBClusterName` then a reference to a `DatabaseClusterBackup` should be called `DBClusterBackupName`.
I can change this if you and @oksana-grishchenko think it's better but in that case I'd also change the  `DBClusterName` we already had in the `DatabaseClusterBackup` for consistency.

Reply:
Let's create a ticket for that and keep it as is